### DISCUSSION TITLE
ScriptReference/AssetDatabase.GenerateUniqueAssetPath

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#697] [Notes]
# ScriptReference/AssetDatabase.GenerateUniqueAssetPath

https://docs.unity3d.com/ScriptReference/AssetDatabase.GenerateUniqueAssetPath.html

**Comment by WispBart** [comment]
If the provided path is incomplete (a folder doesn't exist or hasn't been created yet), this returns an empty string.
